client/src/templates/Introduction/components/challenges.tsx

```diff
@@ -49,40 +52,71 @@ function Challenges({
 
   const isGridMap = isNewRespCert(superBlock) || isNewJsCert(superBlock);
 
+  const firstIncompleteChallenge = challengesWithCompleted.find(
+    challenge => !challenge.isCompleted
+  );
+
   return isGridMap ? (
-    <ul className={`map-challenges-ul map-challenges-grid `}>
-      {challengesWithCompleted.map((challenge, i) => (
-        <li
-          className={`map-challenge-title map-challenge-title-grid ${
-            isProjectBlock ? 'map-project-wrap' : 'map-challenge-wrap'
-          }`}
-          id={challenge.dashedName}
-          key={`map-challenge ${challenge.fields.slug}`}
-        >
-          {!isProjectBlock ? (
-            <Link
-              onClick={() => handleChallengeClick(challenge.fields.slug)}
-              to={challenge.fields.slug}
-              className={`map-grid-item ${
-                challenge.isCompleted ? 'challenge-completed' : ''
+    <>
+      {firstIncompleteChallenge && (
+        <div className='challenge-jump-link'>
+          <Link
+            className='btn btn-primary'
+            onClick={() =>
+              handleChallengeClick(firstIncompleteChallenge.fields.slug)
+            }
+            to={firstIncompleteChallenge.fields.slug}
+          >
+            {t('buttons.resume-project')}{' '}
+            {blockTitle && <span className='sr-only'>{blockTitle}</span>}
+          </Link>
+        </div>
+      )}
+      <nav
+        aria-label={
+          blockTitle ? t('aria.steps-for', { blockTitle }) : t('aria.steps')
+        }
+      >
+        <ul className={`map-challenges-ul map-challenges-grid `}>
+          {challengesWithCompleted.map((challenge, i) => (
+            <li
+              className={`map-challenge-title map-challenge-title-grid ${
+                isProjectBlock ? 'map-project-wrap' : 'map-challenge-wrap'
               }`}
+              id={challenge.dashedName}
+              key={`map-challenge ${challenge.fields.slug}`}
             >
-              {i + 1}
-            </Link>
-          ) : (
-            <Link
-              onClick={() => handleChallengeClick(challenge.fields.slug)}
-              to={challenge.fields.slug}
-            >
-              {challenge.title}
-              <span className=' badge map-badge map-project-checkmark'>
-                {renderCheckMark(challenge.isCompleted)}
-              </span>
-            </Link>
-          )}
-        </li>
-      ))}
-    </ul>
+              {!isProjectBlock ? (
+                <Link
+                  onClick={() => handleChallengeClick(challenge.fields.slug)}
+                  to={challenge.fields.slug}
+                  className={`map-grid-item ${
+                    +challenge.isCompleted ? 'challenge-completed' : ''
+                  }`}
+                >
+                  <span>{i + 1}</span>
```

Comment:
This only reads out '1 link' '2 link' ''3 link' and so on which is not clear to a screen reader user that this means lesson/step 1, lesson/step 2, lesson/step 3 etc. I know it's mentioned in the navigation title but that's a high cognitive load to remember by the time you get to link 67.

Reply:
@emmalearnscode I've gone back and forth on this. Initially I was going to add a screen reader only "Step" before the number but I was trying to keep the verbosity down and also keep the accessible name staring with the the visible text. To compensate, as you pointed out, I put "Steps" in the accessible name for the nav. But I can definitely see the advantages of adding "Step" to make it clearer. We could go with that and then try to get some feedback from screen reader users.

Concerning the cognitive load issue, if adding "Step" before each number would reduce the cognitive load by making it clear that these are links to steps in the course then could the argument be made that we should also add the course name to these links as well for the same reason?

Reply:
I don't think the course name needs adding to each link. I heard it 3 times already by tabbing first the expandable button, then the resume button and then the navigation and when it's a long title it's pretty verbose. I think the cognitive load for me comes with 'one link' having more than one meaning and I interpret it automatically to mean there is one link but it's not being explained to me, not that the link has the text '1' inside it. I think 'step 1 link' removes the ambiguity without adding too much extra verbosity.

Here's what I hear with NVDA (from the expandable button):
Learn HTML by building a Cat Photo App Not started button collapsed heading level 3

Resume project Learn HTML by building a Cat Photo App link

Steps for Learn HTML by building a Cat Photo App Navigation landmark list with 67 items 1 link

2 link

3 link

Reply:
> I interpret it automatically to mean there is one link but it's not being explained to me

Ahh, I see what you are saying. For someone who is completely new to the site I can totally understand that. But I also think most of our users stay for quite a while and would become acquainted with the step navigation list rather quickly and would probably not have this misunderstanding for long. But like I said, I am 50/50 on this, and it is just one extra word, so I'll add it and then if we get any feedback from screen reader users down the road we can address it again.